h/static/scripts/controllers.coffee

```diff
@@ -118,6 +121,9 @@ class AppController
     $scope.clearSelection = ->
       $scope.search.query = ''
       annotationUI.clearSelectedAnnotations()
+      crossframe.notify
```

Comment:
> Something about this seems wrong to me. Specifically, the `annotationUI` service is supposed to be the canonical source of the annotation UI state. That means that we shouldn't have to call two different methods to call in order to clear the selection in both frames.

I agree that it would be nicer if we should only talk to a single component.

> Perhaps the right thing to do is to move the `annotationUI` mutator method into a listener in `annotationUISync`, so that only the crossframe call is here.

To make that work, we would have to teach the host side to echo these messages back to the sidebar. I would evaluate other approaches, too. What about teaching `AnnotatorUI` to transmit the selection info the the host side using the crossframe service?